WEB_SERVICE/WEB/_INFO/PATH_ENUMERATION/description.md

```diff
@@ -0,0 +1 @@
+This information entry aids in identifying Insecure Direct Object Reference (IDOR) vulnerabilities by providing a comprehensive catalog of traversal paths within GraphQL. By examining the various queried resources and potential paths leading to these resources, you can pinpoint areas where authorization should be implemented but may have been overlooked. This systematic approach enhances the detection of potential security gaps in GraphQL implementations.
```

Comment:
I will say something like this information entry provides a holistic overview of different paths to access resources  , This information is  Valuable as it's used  to verify the access control of different objects. this different path can be used latter to find broken access control . (then give an example of two different paths )